gherkin/javascript/src/pretty/pretty.ts

```diff
@@ -5,20 +5,13 @@ export default function pretty(
 ): string {
   const feature = gherkinDocument.feature
   let s = feature.keyword + ': ' + feature.name + '\n'
-  if (feature.description) {
```

Comment:
we should keep the description.

Some search match are done in the description and the tests for filtering should show where the hit was done.